### DISCUSSION TITLE
plan your dive - dive your plan  -> in Dives?

### DISCUSSION THREAD

**Discussion by jro73** [Nelos-VZW/dives-2#86] [Ideas]
Het is eerder een open vraag of bedenking:

Wordt er ook overwogen (voor toekomstige versie) om een duiksessie te plannen in Dives, (voor de duik). 

Dan kan de organisator/verantwoordelijke van de duik de lokatie ingeven + de verschillende teams samenstellen in functie van wie met wie mag duiken (en in welke rol), wie veiligheid kant is.... Deze voorbereiding kan dan ook door "veiligheid kant" gebruikt worden aan te vullen met tijden vertrek en aankomst + bij aankomst diepte en temperatuur.  De duikers moeten naderhand enkel nakijken/bevestigen. 

Vooral voor kwalificaties van duikers en dus wie met wie mag duiken + beperkingen diepte edm is Dives normaal gezien de enige autenthieke bron?  Nu wordt duik naderhand ingevuld en dan pas fout opmerken kan ook zo vermeden worden.. 

Nu verloopt dit proces vaak op papier (of ad hoc) of worden er soms andere tools gebruikt, En alles moet naderhand toch ingegeven worden in Dives.

**Comment by ndsmyter** [comment]
Beetje geschiedenis:
DIVES 1 zoals we die nu kennen, is eigenlijk al een 3e of 4e versie. Maar die eerdere versies zijn nooit uitgebracht. 1 van die versies had inderdaad het idee zoals jij nu voorstelt, dat je op voorhand zaken kon plannen en bekijken. Dit was de versie net voor DIVES 1.0, dus daar blijven nog wel enkele restanten van te zien, als je weet waar je moet kijken.

Maar wat we zagen met die versie, is dat er heel wat werk kruipt om dat allemaal te voorzien, terwijl het voordeel voor NELOS als organisatie heel erg klein was. Want met al die informatie die het aanbood aan de gebruiker, kon het nadien niets doen. Want je kan wel plannen dat Jan een duik gaat doen met Piet, maar ter plaatse kan daar opeens vanalles tussen komen. Dat weet natuurlijk elke persoon die al eens een duikorganisatie gedaan heeft maar al te goed, want je kan uw definitieve indeling pas echt ter plaatse maken.

Als we kijken naar waar NELOS in is geïnteresseerd, dan zijn dat in feite enkel de proeven die afgelegd zijn. Al de rest van duiken en activiteiten zijn in feite maar bijzaak voor NELOS.

Waarmee ik dus wil zeggen: aangezien de meerwaarde voor NELOS om er een planningstool van te maken nihil is, verwacht ik niet dat dit ooit terug zal komen. Dit heeft ook een volledig andere setup dan wat DIVES nu doet, dus daar is dan ook niet heel veel van te gebruiken.
Het voordeel dat het nadien pas gelogd wordt, is dat het onmiddellijk over de feiten kan gaan en niet over een planning.

Ik sluit het niet 100% uit, maar ben toch 90% zeker dat het er waarschijnlijk nooit zal komen :-)
